app/gateway/2.6.x/install-and-run/docker.md

```diff
@@ -151,58 +185,83 @@ docker pull kong:{{page.kong_versions[page.version-index].ce-version}}-alpine
  -p 8443:8443 \
  -p 127.0.0.1:8001:8001 \
  -p 127.0.0.1:8444:8444 \
- kong:latest</code></pre></div>
+ kong:latest
+ ```
 {% endnavtab %}
 {% endnavtabs %}
 {% endcapture %}
 {{ start_container | indent | replace: " </code>", "</code>" }}
 
-   {:.note}
-   > **Note**: The `HOSTNAME` for `KONG_PORTAL_GUI_HOST` should be preceded by a protocol, for example, `http://`.
-
-2. Verify your installation:
+    Where:
+    * `--name` and `--network`: The tag of the {{site.base_gateway}} image that
+    you're using, and the Docker network it communicates on.
+    * [`KONG_DATABASE`](/gateway/{{page.kong_version}}/reference/configuration/#database):
+    Specifies whether this node connects directly to a database.
+    * [`KONG_PG_HOST`](/gateway/{{page.kong_version}}/reference/configuration/#postgres-settings):
+    The name of the Postgres Docker container that is communicating over the
+    `kong-net` network.
+    * [`KONG_PG_USER` and `KONG_PG_PASSWORD`](/gateway/{{page.kong_version}}/reference/configuration/#postgres-settings):
+     The Postgres username and password. Kong Gateway needs the login information
+     to store configurations in this database.
+    * All [`_LOG`](/gateway/{{page.kong_version}}/reference/configuration/#general-section)
+    parameters: set filepaths for the logs to output to, or use the values in
+    the example to  print messages and errors to `stdout` and `stderr`.
+    * [`KONG_ADMIN_LISTEN`](/gateway/{{page.kong_version}}/reference/configuration/#admin_listen):
+    The port that the Kong Admin API listens on for requests.
+    * [`KONG_ADMIN_GUI_URL`](/gateway/{{page.kong_version}}/reference/configuration/#admin_gui_url):
+    (Enterprise only) The URL for accessing Kong Manager, preceded by a protocol
+    (for example, `http://`).
+
+1. Verify your installation:
 
     Access the `/services` endpoint using the Admin API:
 
-    <pre><code>curl -i -X GET --url http://<div contenteditable="true">{HOSTNAME}</div>:8001/services</code></pre>
+    ```sh
+    curl -i -X GET --url http://localhost:8001/services
+    ```
 
     You should receive a `200` status code.
 
-3. (Not available in OSS) Verify that Kong Manager is running by accessing it using the URL specified in `KONG_ADMIN_GUI_URL`:
-
-      <pre><code>http://<div contenteditable="true">{HOSTNAME}</div>:8002</code></pre>
+1. (Not available in OSS) Verify that Kong Manager is running by accessing it using the URL specified in `KONG_ADMIN_GUI_URL`:
 
```

Comment:
If I have a working docker install with a DB here, the next step is to configure a service (link to quickstart?)

I don't want to see the db-less instructions as I already have a gateway

Maybe move `Next Steps` in to this section + the db-less section and duplicate

Reply:
I'd also love instructions on how to stop the containers I just started + remove the network

```bash
docker kill kong-gateway
docker kill kong-database
docker container rm kong-gateway
docker container rm kong-database
docker network rm kong-net
```